docs/03-privacy/privacy-incident-response.md

```diff
@@ -0,0 +1,929 @@
+# Bixal Solutions Privacy Incident Response Plan (IRP)
+
+## Applicability
+
+**Note:**  This Incident Response Plan applies to systems for which Bixal Solutions and third party
+contractors. Individual components may have a tailored version of this Privacy Incident Response
+Plan or in some cases a completely unique Privacy Incident Response Plan will be developed.
+All Bixal Solutions employees are aware of the procedures outlined herein.
+
+## Overview
+
+This Privacy Incident Response Plan provides baseline guidance for the Bixal Solutions Team's
+process for responding to privacy incidents of a Bixal Solutions IRCP
+managed system, product or service ("component"). It outlines roles and
+responsibilities during and after incidents, and it lays out the steps Bixal
+take to resolve them.
+
+## Definitions
+
+**PII** as “any information that permits the identity of an individual to be directly or indirectly
+inferred, including any other information which is linked or linkable to that individual,
+regardless of whether the individual is a United States citizen, lawful permanent resident,
+visitor to the U.S., or employee or contractor to the Department”.
+
+**Sensitive PII** as “personally identifiable information, which if lost, compromised, or disclosed
+without authorization, could result in substantial harm, embarrassment, inconvenience, or
+unfairness to an individual.” Examples of Sensitive PII include, but are not limited to:
+
+1. Social Security numbers (SSN)
+1. drivers license or state identification numbers
+1. passport numbers
+1. Alien Registration numbers
+1. financial account numbers
+1. biometric identifiers
+1. Other data, when combined, may also constitute Sensitive PII, such as:
+   1. citizenship or immigration status
+   1. medical information
+   1. salary
+   1. ethnic or religious affiliation
+   1. personal email address, address, and phone
+   1. account passwords
+   1. date of birth
+   1. criminal history
+   1. mother’s maiden name
+
+## Obligations
+
+As an employee, appointee, detailee, intern, contractor, grantee,
+or consultant (hereafter, Bixal personnel), you have an obligation to report suspected or confirmed
+privacy incidents in a timely manner. By reporting a privacy incident in
```

Comment:
It may be a good idea to add the security email here for reporting an incident?
"By reporting a privacy incident in a timely manner to email@bixal.com, Bixal personnel..."